alivee/utils/requests/exceptions.py

```diff
@@ -0,0 +1,6 @@
+class RequestMethodNotSupportedException(Exception):
+    pass
```

Comment:
@wintonzheng Yes, but it only tells you which line in code raises the exception, but use `RequestMethodNotSupportedException` for example, it cannot tell you which method is causing this exception raised.